superset/__init__.py

```diff
@@ -14,8 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Package's main module!"""
```

Comment:
I removed this silly string to trigger CI as it only runs certain steps based on which folders/files have changed